docs/rules/group-exports.md

```diff
@@ -0,0 +1,62 @@
+# group-exports
+
+Reports when multiple named exports or CommonJS assignments are present in a single file and when the default export is not adjacent to the named export.
```

Comment:
Reworded: _Reports when named exports are not grouped together in a single `export` statement or when multiple assignments to CommonJS `module.exports` or `exports` object are present in a single file._